lab_2_retrieval_w_bm25/start.py

```diff
@@ -26,7 +27,8 @@ def main() -> None:
             documents.append(file.read())
     with open("assets/stopwords.txt", "r", encoding="utf-8") as file:
         stopwords = file.read().split("\n")
-    result = None
+    result = remove_stopwords(tokenize(documents[str]), stopwords)
```

Comment:
what is str??

Reply:
I wanted to make it work, but it does not, so it is not final version of start